backend/src/main/java/com/dhbw/tutorsystem/course/Course.java

```diff
@@ -0,0 +1,41 @@
+package com.dhbw.tutorsystem.course;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name="course") 
+public class Course {
+    
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Getter
+    @Setter
+    private Integer id;
+
+    @Getter
+    @Setter
+    private String description; 
+
+    @Getter
+    @Setter
+    private int year; 
+
+    @Getter
+    @Setter
+    private String 	specialization; 
+
+    @Getter
+    @Setter
+    private String 	courseOfDegree;
```

Comment:
Remove tabs between "String" and the attribute name

Reply:
was solved in [09524b8](https://github.com/FinnHuelsbusch/tutorensystem/pull/12/commits/09524b86d247fef36caf37bb4e3d0e135a82737b)